arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -705,48 +824,56 @@
           "description": "Diagnostics storage account table endpoint",
           "type": "string"
         }
-      }
+      },
+      "required": [
+        "storageAccountName"
+      ]
     },
     "ClusterPropertiesUpdateParameters": {
-      "description": "The cluster resource properties",
+      "description": "The cluster resource properties can be updated",
       "properties": {
         "reliabilityLevel": {
-          "description": "Cluster reliability level",
+          "description": "Cluster reliability level indicates replica set size of system service",
           "type": "string",
           "enum": [
-            "Invalid",
             "Bronze",
             "Silver",
             "Gold",
             "Platinum"
           ]
         },
         "upgradeMode": {
-          "description": "Cluster upgrade mode",
+          "description": "Cluster upgrade mode indicates if fabric upgrade is initiated automatically by the system or not",
           "type": "string",
           "enum": [
-            "Default",
             "Automatic",
             "Manual"
           ]
         },
         "clusterCodeVersion": {
-          "description": "The ServiceFabric code version, if set it, please make sure you have set upgradeMode to Manual, otherwise ,it will fail",
+          "description": "The ServiceFabric code version, if set it, please make sure you have set upgradeMode to Manual, otherwise ,it will fail, if you are using PUT new cluster,  you can get the version by using ClusterVersions_List , if you are updating existing cluster, you can get the availableClusterVersions from ClusterGetOperation,  ",
           "type": "string"
         },
         "certificate": {
-          "description": "The cluster certificate settings, the new certificate should exist in Vmss or KeyVault,before you add it, it will override original value",
+          "description": "The cluster certificate settings, the new certificate should exist in VMSS or Azure key vault,before you add it. It will override original value",
```

Comment:
VMSS = Vmss, earlier was right only 2 letters acronyms are supposed to be all caps. Better to say virtual machine scale sets, rather than saying acronym (check with your PM for this).